bundles/AdminBundle/Resources/public/js/pimcore/asset/video.js

```diff
@@ -111,120 +117,147 @@ pimcore.asset.video = Class.create(pimcore.asset.asset, {
 
             var date = new Date();
 
-            this.previewImagePanel = new Ext.Panel({
-                width: 300,
-                region: "east",
-                scrollable: 'y',
+            var detailsData = [];
```

Comment:
We should check whether the data is available or not, if no FFMPEG is installed, this data isn't there.